src/index.ts

```diff
@@ -24,11 +24,26 @@ async function run() {
     })
   )
 
-  // 3) Post comment
   if (!inputs.skipEmpty || !planIsEmpty(plan)) {
-    await core.group('Render comment', () => {
-      const comment = renderComment({ plan, header: inputs.header })
-      return createOrUpdateComment({ octokit, content: comment })
+    const prContext = 'pull_request' in github.context.payload
```

Comment:
When writing GitHub action workflows, I typically use `github.event_name == 'pull_request'`. I think the equivalent check here would be:

```suggestion
    const prContext = github.context.eventName === 'push'
```